dbms/src/Common/StringUtils/StringUtils.h

```diff
@@ -155,6 +160,18 @@ inline bool isPunctuation(char c)
     return (c >= 0x21 && c <= 0x2F) || (c >= 0x3A && c <= 0x40) || (c >= 0x5B && c <= 0x60) || (c >= 0x7B && c <= 0x7E);
 }
 
+inline size_t getUtf8Length(uint8_t c)
+{
+    size_t size = 0;
+    while (c >= 128)
+    {
+        if ((c & 128) == 128)
+            ++size;
+        c <<= 1;
+    };
+    return size;
+}
```

Comment:
add some comments about this function? And IMO, `UTF8` is better than `Utf8`.

Reply:
> add some comments about this function? And IMO, `UTF8` is better than `Utf8`.

okk